charts/dapr/charts/dapr_dashboard/templates/dapr_dashboard_deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dapr-dashboard
+  labels:
+    app: dapr-dashboard
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: dapr-dashboard
+  template:
+    metadata:
+      labels:
+        app: dapr-dashboard
+    spec:
+      containers:
+      - name: dapr-dashboard
+        image: docker.io/daprio/dashboard:0.1.0
```

Comment:
this can be:
image: "{{ .Values.global.registry }}/{{ .Values.image.name }}:{{ .Values..tag }}"

charts/dapr/charts/dapr_dashboard/values.yaml will have the tag defined in it as 0,1,0, this allows inf future to only update values.yanl file for new dashboard releases.
tag: "0.1.0"